examples/00-Overview/02-get_data_from_static_simulation.py

```diff
@@ -0,0 +1,97 @@
+"""
+.. _ref_get_data_from_static_simulation:
+
+Get data from static simulation
+===============================
+This example show how to request data from a previously stored static simulation.
+The available results can be listed to see what results can be retrieved.
+"""
+
+###############################################################################
+# Imports and loading simulation
+# ------------------------------
+from ansys.dpf.post import examples, load_simulation
+
+simulation = load_simulation(examples.download_all_kinds_of_complexity())
+
+###############################################################################
+# Get and plot displacements
+# --------------------------
+disp = simulation.displacement()
```

Comment:
If possible, please use `displacement` instead of `disp` in the example, except if this is the name of a DPF module that could create confusion for users.
```suggestion
displacement = simulation.displacement()
```

Reply:
Done!